ui/apps/platform/src/Containers/ComplianceEnhanced/ScanConfigs/Table/ScanConfigsTablePage.tsx

```diff
@@ -1,98 +1,248 @@
-    }
-
-    function fetchScanSchedules(query: string) {
-        setIsLoading(true);
-        getScanConfigs(query)
-            .then((data) => {
-                setScanSchedules(data);
-                setErrorMessage('');
-            })
-            .catch((error) => {
-                setScanSchedules([]);
-                setErrorMessage(getAxiosErrorMessage(error));
-            })
-            .finally(() => setIsLoading(false));
-    }
-
-    const query = searchFilter ? getRequestQueryStringForSearchFilter(searchFilter) : '';
-
-    useEffect(() => {
-        fetchScanSchedules(query);
-    }, [query]);
-
-    let pageContent = (
-        <PageSection variant="light" isFilled id="policies-table-loading">
-            <Bullseye>
-                <Spinner isSVG />
-            </Bullseye>
-        </PageSection>
+    const { page, perPage, setPage, setPerPage } = useURLPagination(10);
+    const { sortOption, getSortParams } = useURLSort({
+        sortFields,
+        defaultSortOption,
+    });
+
+    const listQuery = useCallback(
+        () => getScanConfigs(sortOption, page - 1, perPage),
+        [sortOption, page, perPage]
     );
+    const { data: scanSchedules, loading: isLoading, error } = useRestQuery(listQuery);
+
+    const formatScanSchedule = (schedule: Schedule) => {
+        const daysOfWeekMap = ['Sun', 'Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat'];
+
+        const formatDays = (days: string[]): string => {
+            if (days.length === 1) {
+                return days[0];
+            }
+            if (days.length === 2) {
+                return days.join(' and ');
+            }
+            return `${days.slice(0, -1).join(', ')}, and ${days[days.length - 1]}`;
+        };
+
+        // arbitrary date, we only care about the time
+        const date = new Date(2000, 0, 0, schedule.hour, schedule.minute);
+        const timeString = getTimeHoursMinutes(date);
+
+        switch (schedule.intervalType) {
+            case 'DAILY':
+                return `Daily at ${timeString}`;
+            case 'WEEKLY':
+                if (schedule.daysOfWeek) {
+                    const daysOfWeek = schedule.daysOfWeek.days.map((day) => daysOfWeekMap[day]);
+                    return `Every ${formatDays(daysOfWeek)} at ${timeString}`;
+                }
+                break;
+            case 'MONTHLY':
+                if (schedule.daysOfMonth) {
+                    const formattedDaysOfMonth =
+                        schedule.daysOfMonth.days.map(getDayOfMonthWithOrdinal);
+                    return `Monthly on the ${formatDays(formattedDaysOfMonth)} at ${timeString}`;
+                }
+                break;
+            default:
+                return 'Invalid Schedule';
+        }
 
-    if (errorMessage) {
-        pageContent = (
-            <PageSection variant="light" isFilled id="policies-table-error">
+        return 'Invalid Schedule';
```

Comment:
I believe there's actually a lint error without it because of the `if` statements within the `WEEKLY` and `MONTHLY` cases. Although now that I'm looking at it, I thought I set up the typing to where if either of those intervals are used then the `daysOfWeek`/`daysOfMonth` are required. So those `if` statements might not be needed.